code/script.js

```diff
@@ -1,19 +1,100 @@
+  foodName = "Pasta";
+} else if (choosefood === "3") {
+  foodName = "Salad";
+} else {
+  alert(`Sorry, invalid number/choice. Please start over.`); process.exit(1);
+}
+
+alert("Yummy! " + foodName + "! What a great choice!");
+
+// Step 3 - Subtype choice depending on food choice
+let specifyfood;
+
+if (foodName === "Pizza") {
+  specifyfood = prompt("Select a type of pizza: 1 = Spicy pizza, 2 = Yummy Pizza, 3= Yummy yummy pizza")
+} else if (foodName === "Pasta") {
+  specifyfood = prompt("Select a type of pasta: 1 = Spicy spaghetti, 2 = Yummy spagetthi, 3 = Yummy yummy spagetthi", "Type your number of choice here");
+} else if (foodName === "Salad") {
+  specifyfood = prompt("Select a type of salad: 1 = Spicy salad, 2 = Yummy salad, 3 = Yummy yummy salad", "Type your number of choice here");
+} else {
+  alert(`Sorry, invalid choice. Please start over.`); process.exit(1);
+}
+
+//Determining the chosen subtype and providing feedback using alert method to confirm the selection.
+let finalchoice
+
+if (foodName === "Pizza") {
+  if (specifyfood === "1") {
+    finalchoice = "Spicy pizza"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  } else if (specifyfood === "2") {
+    finalchoice = "Yummy pizza"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  }
+  else if (specifyfood === "3") {
+    finalchoice = "Yummy yummy pizza"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  }
+} else if (foodName === "Pasta") {
+  if (specifyfood === "1") {
+    finalchoice = "Spicy spagetthi"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  } else if (specifyfood === "2") {
+    finalchoice = "Yummy spagetthi"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  }
+  else if (specifyfood === "3") {
+    finalchoice = "Yummy yummy spagetthi"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  }
+} else if (foodName === "Salad") {
+  if (specifyfood === "1") {
+    finalchoice = "Spicy salad"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  } else if (specifyfood === "2") {
+    finalchoice = "Yummy salad"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  }
+  else if (specifyfood === "3") {
+    finalchoice = "Yummy yummy salad"
+    alert(`You choosed the ${foodName}: ${finalchoice}`);
+  }
+} else {
+  alert(`Invalid choice. Please start over.`); process.exit(1);
+}
 
-// Step 3 - Subtype choice
-// Your code goes here
+// Step 4 and 5 - Age. Adolescent/adult or child. And confirmation.
+let age = prompt("Portion sizes differ depending on if it's a child or adolescent/adult.", "Please type your age here");
+let confirmation;
 
-// Step 4 - Age
-// Your code goes here
+if (age >= 16) {
+  confirmation = prompt(`One regular sized ${finalchoice}. That'll be 10 euros.`, `Enter a number to confirm: 1: Yes, 2: No.`);
+} else if (age <= 15) {
+  confirmation = prompt(`One child sized ${finalchoice}. That'll be 7 euros.`, `Enter a number to confirm: 1: Yes, 2: No.`);
+}
```

Comment:
This works! But I think it also would have been enough to just do an if-else where you only specify when it's for a child. Seeing as you only have two options, adult and child, you don't need to specify when it's for an adult because that is the case for every age after 16. 

```suggestion
if (age <= 16) {
  confirmation = prompt(`One child sized ${finalchoice}. That'll be 7 euros.`, `Enter a number to confirm: 1: Yes, 2: No.`);
} else {
  confirmation = prompt(`One regular sized ${finalchoice}. That'll be 10 euros.`, `Enter a number to confirm: 1: Yes, 2: No.`);
}
```

Reply:
Ah! Thank you!